src/Keboola/DbWriter/Redshift/Application.php

```diff
@@ -8,24 +8,57 @@
 
 namespace Keboola\DbWriter\Redshift;
 
-use \Keboola\DbWriter\Application as BaseApplication;
+use Keboola\DbWriter\Configuration\Validator;
 use Keboola\DbWriter\Exception\ApplicationException;
 use Keboola\DbWriter\Exception\UserException;
+use Keboola\DbWriter\Logger;
+use Keboola\DbWriter\Redshift\Configuration\ConfigDefinition;
 use Keboola\DbWriter\Writer\Redshift;
-use Symfony\Component\Yaml\Yaml;
+use Pimple\Container;
 
-class Application extends BaseApplication
+class Application
```

Comment:
jakej je duvod pro vyhozeni te `BaseApplication`

Reply:
V podstate ziadny, to som sa trochu premotivoval, ze chcem mat ten kontajner inac a chcel som do neho pridat z konfigu ten input mapping, co by som mohol vyriesit pretazenim konstruktoru. To by asi bolo lepsie. Najprv som chcel mat istotu, ze sa tam nedeje nieco podvine v parentovi.